src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let relationshipStatus = '';
+
+  if (!isMarried) {
+    relationshipStatus = 'I am not married';
+  } else {
+    const partnerRole = sex === 'f' ? 'husband' : 'wife';
```

Comment:
Try not to combine if statement with ternary, you can just do smth like:
 1. Create a variable for partner:
 `const partner = sex === 'f' ? 'husband' : 'wife';`
 2. Add this condition as a separate variable or directly to jsx
         isMarried
          ? `${partnerName} is my ${partner}`
          : 'I am not married'